conf/layer.conf

```diff
@@ -11,4 +11,9 @@ BBFILE_PRIORITY_celebrity-face-match = "24"
 LAYERSERIES_COMPAT_celebrity-face-match = "kirkstone"
 LAYERDEPENDS_celebrity-face-match = "phytec ampliphy meta-ml"
 
+BBMASK += "meta-ampliphy/recipes-graphics/wayland/weston-init.bbappend"
+BBMASK += "meta-imx/meta-bsp/recipes-graphics/wayland/weston-init.bbappend"
+BBMASK += " meta-freescale/recipes-graphics/wayland/weston-init.bbappend"
+BBMASK += " meta-freescale-distro/recipes-graphics/wayland/weston-init.bbappend"
```

Comment:
Are these mixes of leading whitespaces intentional?

Reply:
Good point, I guess not. The latter two were added by me. Will fix.